### ISSUE TITLE
How to run test script?

### ISSUE THREAD

**Issue by sumeshthakr** [2226171237/MyS2VT#3]
How to test the model with test videos?

**Comment by 2226171237** [comment]
I did not split the test set from the dataset. please refer to line 108 of s2vt. py

**Comment by sumeshthakr** [comment]
Thanks for the reply,

Is there a way to test the model on a unseen (not a part of train/test dataset) video directly?

**Comment by 2226171237** [comment]
I have written a new test script (generator.py). Please try the generator.py script. I hope this is what you want.

**Comment by sumeshthakr** [comment]
Thanks a lot, your repo is very helpful,

Trying to implement in pytorch,
